src/installation/guides/fde.md

```diff
@@ -0,0 +1,254 @@
+# Full Disk Encryption
+
+Your drive's block device and other information may be different, so make sure
+it is correct.
+
+Boot the live image and login.
+
+Create a single physical partition on the disk using `cfdisk`, marking it
+bootable. For an MBR system, the partition layout should look the following.
+
+```
+# fdisk -l /dev/sda
+Disk /dev/sda: 48 GiB, 51539607552 bytes, 100663296 sectors
+Units: sectors of 1 * 512 = 512 bytes
+Sector size (logical/physical): 512 bytes / 512 bytes
+I/O size (minimum/optimal): 512 bytes / 512 bytes
+Disklabel type: dos
+Disk identifier: 0x4d532059
+
+Device     Boot Start       End   Sectors Size Id Type
+/dev/sda1  *     2048 100663295 100661248  48G 83 Linux
+```
+
+UEFI systems will need the disk to have a GTP disklabel and an EFI system
+partition. The required size for this may vary depending on needs, but 100M
+should be enough for most cases. For an EFI system, the partition layout should
+look the following.
+
+```
+# fdisk -l /dev/sda
+Disk /dev/sda: 48 GiB, 51539607552 bytes, 100663296 sectors
+Units: sectors of 1 * 512 = 512 bytes
+Sector size (logical/physical): 512 bytes / 512 bytes
+I/O size (minimum/optimal): 512 bytes / 512 bytes
+Disklabel type: gpt
+Disk identifier: EE4F2A1A-8E7F-48CA-B3D0-BD7A01F6D8A0
+
+Device      Start       End   Sectors  Size Type
+/dev/sda1    2048    264191    262144  128M EFI System
+/dev/sda2  264192 100663262 100399071 47.9G Linux filesystem
+```
+
+Configure the encrypted volume.
+
+```
+# cryptsetup luksFormat --type luks1 /dev/sda1
```

Comment:
Why did you default to luks1? I think nowadays most kernels in Void support luks2, and it is a better option, if supported. This could perhpas be explained.

Reply:
grub doesn't support luks2 yet. 

Support has been merged, but it hasnt been released

Reply:
Once it's released it isn't hard to convert to luks2